--- memcache-client-1.8.5/test/test_mem_cache.rb.orig	2025-02-25 15:53:32.417036214 +0100
+++ memcache-client-1.8.5/test/test_mem_cache.rb	2025-02-25 15:53:50.040520297 +0100
@@ -1,16 +1,9 @@
 # encoding: utf-8
-require 'rubygems'
 require 'logger'
 require 'stringio'
 require 'test/unit'
 $TESTING = true
 require 'memcache'
-
-begin
-  gem 'flexmock'
-  require 'flexmock/test_unit'
-rescue LoadError => e
-  puts "Some tests require flexmock, please run `gem install flexmock`"
-end
+require 'flexmock/test_unit'
 
 Thread.abort_on_exception = true
